client/js/upload-handler/upload.handler.controller.js

```diff
@@ -590,6 +584,7 @@ qq.UploadHandlerController = function(o, namespace) {
     };
 
     qq.extend(this, {
+        connectionManager: connectionManager,
```

Comment:
I'm not sure I like making `connectionManager` in its entirely available to all code outside of this module. I really don't think anything other than `connectionManager` should touch its internal state.